ENGAGEHF/ENGAGEHFStandard.swift

```diff
@@ -94,6 +108,16 @@ actor ENGAGEHFStandard: Standard, EnvironmentAccessible, OnboardingConstraint, A
     }
     
     
+    func add(symptomScore: SymptomScore) async {
+        do {
+            let userDoc = try await userDocumentReference
```

Comment:
User Document Reference should be renamed to `patientDocumentReference` and updated in accordance to the changes that @pauljohanneskraft merges today to ideally not break the client.